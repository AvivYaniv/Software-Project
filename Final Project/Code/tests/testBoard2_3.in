solve tests/b2_3.txt
print_board
autofill
set 4 5 3
set 4 6 5
undo
redo
set 3 5 1
num_solutions
hint 5 5
hint 6 5
set 5 6 2
mark_errors 0
print_board
mark_errors 1
print_board
undo
validate
save tests/observed/b2_3-saveTestSolve.txt
edit tests/observed/b2_3-saveTestSolve.txt
print_board
set 3 4 1
set 3 5 4
undo 
undo
save tests/observed/b2_3-saveTestEdit.txt
solve tests/observed/b2_3-saveTestSolve.txt
print_board
set 3 4 1
set 3 5 4