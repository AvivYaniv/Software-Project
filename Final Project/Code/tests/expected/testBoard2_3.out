Sudoku
------
Enter your command:
---------------------------
|  1.  2.  3.|  4.  5.  6.|
|  4.  5.  6.|  1.  2.  3.|
---------------------------
|  2.  1.  4.|  6.  3.  5.|
|  3.  6.  5.|            |
---------------------------
|  5.  4.    |            |
|  6.        |            |
---------------------------
Enter your command:
---------------------------
|  1.  2.  3.|  4.  5.  6.|
|  4.  5.  6.|  1.  2.  3.|
---------------------------
|  2.  1.  4.|  6.  3.  5.|
|  3.  6.  5.|            |
---------------------------
|  5.  4.    |            |
|  6.        |            |
---------------------------
Enter your command:
Cell <4,4> set to 2
Cell <2,6> set to 3
---------------------------
|  1.  2.  3.|  4.  5.  6.|
|  4.  5.  6.|  1.  2.  3.|
---------------------------
|  2.  1.  4.|  6.  3.  5.|
|  3.  6.  5.|  2         |
---------------------------
|  5.  4.    |            |
|  6.  3     |            |
---------------------------
Enter your command:
---------------------------
|  1.  2.  3.|  4.  5.  6.|
|  4.  5.  6.|  1.  2.  3.|
---------------------------
|  2.  1.  4.|  6.  3.  5.|
|  3.  6.  5.|  2         |
---------------------------
|  5.  4.    |  3         |
|  6.  3     |            |
---------------------------
Enter your command:
---------------------------
|  1.  2.  3.|  4.  5.  6.|
|  4.  5.  6.|  1.  2.  3.|
---------------------------
|  2.  1.  4.|  6.  3.  5.|
|  3.  6.  5.|  2         |
---------------------------
|  5.  4.    |  3         |
|  6.  3     |  5         |
---------------------------
Enter your command:
---------------------------
|  1.  2.  3.|  4.  5.  6.|
|  4.  5.  6.|  1.  2.  3.|
---------------------------
|  2.  1.  4.|  6.  3.  5.|
|  3.  6.  5.|  2         |
---------------------------
|  5.  4.    |  3         |
|  6.  3     |            |
---------------------------
Undo 4,6: from 5 to _
Enter your command:
---------------------------
|  1.  2.  3.|  4.  5.  6.|
|  4.  5.  6.|  1.  2.  3.|
---------------------------
|  2.  1.  4.|  6.  3.  5.|
|  3.  6.  5.|  2         |
---------------------------
|  5.  4.    |  3         |
|  6.  3     |  5         |
---------------------------
Redo 4,6: from _ to 5
Enter your command:
---------------------------
|  1.  2.  3.|  4.  5.  6.|
|  4.  5.  6.|  1.  2.  3.|
---------------------------
|  2.  1.  4.|  6.  3.  5.|
|  3.  6.  5.|  2         |
---------------------------
|  5.  4.  1 |  3         |
|  6.  3     |  5         |
---------------------------
Enter your command:
Number of solutions: 2
The puzzle has more than 1 solution, try to edit it further
Enter your command:
Hint: set cell to 6
Enter your command:
Hint: set cell to 2
Enter your command:
---------------------------
|  1.  2.  3.|  4.  5.  6.|
|  4.  5.  6.|  1.  2.  3.|
---------------------------
|  2.  1.  4.|  6.  3.  5.|
|  3.  6.  5.|  2         |
---------------------------
|  5.  4.  1 |  3         |
|  6.  3     |  5   2*    |
---------------------------
Enter your command:
Enter your command:
---------------------------
|  1.  2.  3.|  4.  5.  6.|
|  4.  5.  6.|  1.  2.  3.|
---------------------------
|  2.  1.  4.|  6.  3.  5.|
|  3.  6.  5.|  2         |
---------------------------
|  5.  4.  1 |  3         |
|  6.  3     |  5   2     |
---------------------------
Enter your command:
Enter your command:
---------------------------
|  1.  2.  3.|  4.  5.  6.|
|  4.  5.  6.|  1.  2.  3.|
---------------------------
|  2.  1.  4.|  6.  3.  5.|
|  3.  6.  5.|  2         |
---------------------------
|  5.  4.  1 |  3         |
|  6.  3     |  5   2*    |
---------------------------
Enter your command:
---------------------------
|  1.  2.  3.|  4.  5.  6.|
|  4.  5.  6.|  1.  2.  3.|
---------------------------
|  2.  1.  4.|  6.  3.  5.|
|  3.  6.  5.|  2         |
---------------------------
|  5.  4.  1 |  3         |
|  6.  3     |  5         |
---------------------------
Undo 5,6: from 2 to _
Enter your command:
Validation passed: board is solvable
Enter your command:
Saved to: tests/observed/b2_3-saveTestSolve.txt
Enter your command:
---------------------------
|  1   2   3 |  4   5   6 |
|  4   5   6 |  1   2   3 |
---------------------------
|  2   1   4 |  6   3   5 |
|  3   6   5 |  2         |
---------------------------
|  5   4   1 |  3         |
|  6   3     |  5         |
---------------------------
Enter your command:
---------------------------
|  1   2   3 |  4   5   6 |
|  4   5   6 |  1   2   3 |
---------------------------
|  2   1   4 |  6   3   5 |
|  3   6   5 |  2         |
---------------------------
|  5   4   1 |  3         |
|  6   3     |  5         |
---------------------------
Enter your command:
---------------------------
|  1   2   3 |  4   5   6 |
|  4   5   6 |  1   2   3 |
---------------------------
|  2   1*  4 |  6   3   5 |
|  3   6   1*|  2         |
---------------------------
|  5   4   1*|  3         |
|  6   3     |  5         |
---------------------------
Enter your command:
---------------------------
|  1   2   3 |  4   5   6 |
|  4   5   6 |  1   2   3 |
---------------------------
|  2   1*  4*|  6   3   5 |
|  3   6   1*|  2         |
---------------------------
|  5   4*  4*|  3         |
|  6   3     |  5         |
---------------------------
Enter your command:
---------------------------
|  1   2   3 |  4   5   6 |
|  4   5   6 |  1   2   3 |
---------------------------
|  2   1*  4 |  6   3   5 |
|  3   6   1*|  2         |
---------------------------
|  5   4   1*|  3         |
|  6   3     |  5         |
---------------------------
Undo 3,5: from 4 to 1
Enter your command:
---------------------------
|  1   2   3 |  4   5   6 |
|  4   5   6 |  1   2   3 |
---------------------------
|  2   1   4 |  6   3   5 |
|  3   6   5 |  2         |
---------------------------
|  5   4   1 |  3         |
|  6   3     |  5         |
---------------------------
Undo 3,4: from 1 to 5
Enter your command:
Saved to: tests/observed/b2_3-saveTestEdit.txt
Enter your command:
---------------------------
|  1.  2.  3.|  4.  5.  6.|
|  4.  5.  6.|  1.  2.  3.|
---------------------------
|  2.  1.  4.|  6.  3.  5.|
|  3.  6.  5.|  2         |
---------------------------
|  5.  4.  1 |  3         |
|  6.  3     |  5         |
---------------------------
Enter your command:
---------------------------
|  1.  2.  3.|  4.  5.  6.|
|  4.  5.  6.|  1.  2.  3.|
---------------------------
|  2.  1.  4.|  6.  3.  5.|
|  3.  6.  5.|  2         |
---------------------------
|  5.  4.  1 |  3         |
|  6.  3     |  5         |
---------------------------
Enter your command:
Error: cell is fixed
Enter your command:
---------------------------
|  1.  2.  3.|  4.  5.  6.|
|  4.  5.  6.|  1.  2.  3.|
---------------------------
|  2.  1.  4.|  6.  3.  5.|
|  3.  6.  5.|  2         |
---------------------------
|  5.  4.  4*|  3         |
|  6.  3     |  5         |
---------------------------
Enter your command:
Exiting...
